source/environments/Pict-Environment-Object.js

```diff
@@ -203,6 +218,41 @@ class PictEnvironmentObject
 
 		return true;
 	}
+
+	customReadClassFunction (pAddress, pClass)
+	{
+		let tmpContent = this.contentMap._CLASS_LIST.indexOf(pClass) > -1 ? true : false;
+
+		if (this.storeEventLog) this.eventLog.Assign.push(this.createEventLogEntry(pAddress, pClass));
+		this.pict.log.info(`Mocking an CLASS READ for Address -> [${pAddress}]::[${pClass}]`, {Content: tmpContent});
+		return tmpContent;
+	}
+
+	customSetClassFunction (pAddress, pClass)
+	{
+		let tmpContent = this.contentMap._CLASS_LIST;
+
+		if (!this.contentMap._CLASS_LIST.indexOf(pClass) > -1)
+		{
+			this.contentMap._CLASS_LIST.push(pClass);
+		}
+
+		if (this.storeEventLog) this.eventLog.Assign.push(this.createEventLogEntry(pAddress, pClass));
+		this.pict.log.info(`Mocking an CLASS SET for Address -> [${pAddress}]::[${pClass}]`, {Content: tmpContent.join()});
+		return tmpContent;
```

Comment:
because _CLASS_LIST is a list, this mock is adding the class for "all" addresses/elements.

I think we want a _CLASS_MAP instead.

Reply:
Updated.